WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -315,7 +315,9 @@ class GutenbergViewController: UIViewController, PostEditor, FeaturedImageDelega
 
         addObservers(toPost: post)
 
-        PostCoordinator.shared.cancelAnyPendingSaveOf(post: post)
+        if !RemoteFeatureFlag.syncPublishing.enabled() {
+            PostCoordinator.shared.cancelAnyPendingSaveOf(post: post)
```

Comment:
We no longer want the editor to cancel the pending uploads: `PostCoordinator` _supports_ background uploads.